kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue

```diff
@@ -30,44 +30,72 @@
     @timeupdate="timeUpdate"
     @loadedmetadata="setTotalTime"
     v-el:audio 
-    src="http://www.stephaniequinn.com/Music/Commercial%20DEMO%20-%2004.mp3">
-  </audio>
+    src="http://www.stephaniequinn.com/Music/Commercial%20DEMO%20-%2004.mp3"
+    ></audio>
 
 </template>
 
 
 <script>
 
+  const logging = require('loglevel');
   require('html5media/dist/api/1.1.8/html5media');
   module.exports = {
 
     data: () => ({
-      isPlay: true,
-      isPause: false,
-      timebarChanged: false,
-      currentMinutes: 0,
-      currentSeconds: 0,
-      totalMinutes: 0,
-      totalSeconds: 0,
+      isPlay: {
+        type: Boolean,
+        status: true,
+      },
+      isPause: {
+        type: Boolean,
+        status: false,
+      },
+      timebarChanged: {
+        type: Boolean,
+        status: false,
+      },
+      currentMinutes: {
+        type: Number,
+        min: 0,
+      },
+      currentSeconds: {
+        type: Number,
+        sec: 0,
+      },
+      totalMinutes: {
+        type: Number,
+        totalmin: 0,
+      },
+      totalSeconds: {
+        type: Number,
+        totalsec: 0,
+      },
```

Comment:
Hey - looks like you may be applying [props-style validation](https://vuejs.org/guide/components.html#Prop-Validation) to the `data` attributes? I don't believe it works the same way for `data` - as far as I know vue doesn't type-check or validate these values.